command/operator_migrate.go

```diff
@@ -98,6 +99,14 @@ func (c *OperatorMigrateCommand) Flags() *FlagSets {
 		Usage:  "Reset the migration lock. No migration will occur.",
 	})
 
+	f.IntVar(&IntVar{
```

Comment:
If we're doing checks at the start of the func, then it's not so much of an issue, but for clarity perhaps you could consider using `UintVar` as this would eliminate anyone doing something silly like passing `-parallel=-10` etc.